### ISSUE TITLE
Using TypeScript complains about CommonsJS modules found in Variables folder

### ISSUE THREAD

**Issue by flackjap** [creativetimofficial/ct-light-bootstrap-dashboard-pro-react#41]
# Expected Behavior

Using TypeScript with `npx create-react-app my-app --typescript` should not raise any errors or warnings, but it does - it's not compatible with CommonJS modules that can be found in Variables folder.

# Current Behavior

As far as I could understand, TypeScript fails to recognize that the "module" as an object is a reserved syntax for CommonJS modules, and that you can define "exports" on it. Besides that, its compiler also doesn't recognize that it's trying to export anything.

## Steps to Reproduce

1. Install React with `npx create-react-app my-app --typescript`
2. Copy all LBD folders found in src (just folders, not the root files)
3. Copy routes.js from LBD to the src folder of the newly created React app
4. Copy public folder and its files from LBD
5. Copy everyhing from jsconfig.json into the respective compilerOptions of tsconfig.json
6. Copy all imports from index.js into the App.tsx
7. Manually merge the LBD's package.json with the one created by create-react-app.
8. npm i @types/react-router-dom
9. npm install (or yarn install)
10. npm start

p.s. Maybe it will complain about some additional types too. I just remember installing the @types/react-router-dom manually (mentioned at the previous 8th step).

## Context

* Operating System: Doesn't matter. Works same on OSX and Debian Linux.
Node version - v12.2.0

Versions of relevant packages:
* "@types/node": "12.0.1"
* "typescript": "3.4.5"
* "react": "16.8.6"

## Failure Logs

If you don't touch anything, you get:
```
Failed to compile
./src/views/Charts.jsx
Attempted import error: 'charts' is not exported from 'variables/chartsVariables.jsx'.
```

If you just add exports below the existing module.exports, you get:
` TypeError: Cannot assign to read only property 'exports' of object '#<Object>' `
Answer for the related issue: https://github.com/webpack/webpack/issues/4039#issuecomment-273804003

So, just by rewriting module.exports to the plain ES6 exports, it compiles, but I'm not sure if that would break regular React app without TS - didn't have time to try that, but I guess that it should work.

I could see that the rest of the code around LBD's React is using ES6 module syntax. Why are these Variable files still using CommonJS ?

**Comment by einazare** [comment]
Hello there, @flackjap ,

Thank you for your interest in working with our products and thank you for sharing your solution.
I remember that when I created this product, I had the same issues from here: https://stackoverflow.com/questions/34645731/export-more-than-one-variable-in-es6/34645759
The problem was, that I couldn't export multiple variables/functions with ES6. I know that I've tried multiple solutions found on the internet an nothing really worked.
In the next update, I'll make the necessary changes.

Once again, thank you for raising this issue.

All the best,
Manu
